components/scheduler/src/Scheduler.svelte

```diff
@@ -27,10 +37,13 @@
   export let event_description: string;
   export let event_location: string;
   export let event_conferencing: string;
-  export let slots_to_book: TimeSlot[] = [];
+  export let slots_to_book: BookableSlot[] = [];
   export let notification_mode: NotificationMode;
   export let notification_message: string;
   export let notification_subject: string;
+  export let recurrence: "none" | "mandated" | "optional";
```

Comment:
Perhaps `"mandated"` can be changed to `"required"`? I think that might be a bit clearer?

Reply:
Good catch; both issues now fixed